Missing headers Bug Fix in ThreatCrowdv2 (#13223)
demisto/content: Packs/Threat_Crowd/ReleaseNotes/2_0_4.md

#### Integrations
##### Threat Crowd v2

> - Fixed a bug where command failed due to 'references' data in result. 

Reviewer: please always try to curate a rn that explains what was fixed in a manner that a non dev would comprehend, e.g:
```suggestion
- Fixed a bug where the ***ip*** command failed on certain values. 
```
